app/assets/javascripts/contacts.js

```diff
@@ -72,8 +85,101 @@ var processContacts = function(contacts, status, xhr) {
   });
 };
 
-var showContactInForm = function(position) {
+var showEmptyContactList = function() {
+  $(".contacts").append(
+    "<li data-role='contact'>No contacts yet</li>"
+  );
+};
+
+var processContacts = function(contacts, status, xhr) {
```

Comment:
We use the convention of using leading underscores for variables you don't use (`(contacts, _status, _xhr)`). But in JS I typically just leave them out unless I need one later in the chain (I need xhr and I don't need status for example).